### PR TITLE
Complete Week 5

### PR DESCRIPTION
**Pull request by lundrew**

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -23,12 +32,52 @@ const secretCodeWord3 = "Eccentric"
...
+const coded = (string) => {
+
+    let newstring = string.replace(/a|A/g, "4")
+    let newstring2 = newstring.replace(/e|E/gi, "3")
+    let newstring3 = newstring2.replace(/i|I/gi, "1")
+    let newstring4 = newstring3.replace(/o|O/gi, "0")
+
+    return newstring4
+    
+  }
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:65]: Nice! Good use of variables to track your progress.

File: code-challenges.test.js [hunk 2]
```diff
@@ -38,7 +87,44 @@ const letterE = "e"
...
+// create a function called "specific"
+// create 2 params named "array" and "letter"
+// return the array that uses the filter() method 
+// verify the string includes a specific letter using the includes() method on the "letter" parameter and use an "or" to check uppercase letters.
+
+const specific = (array, letter) => {
+
+    return array.filter(string => { return string.includes(letter) || string.includes(letter.toUpperCase())}) 
+
+   }
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:115]: Very clean, good catch on edge cases.

File: code-challenges.test.js [hunk 3]
```diff
@@ -52,5 +138,57 @@ const hand2 = [5, 5, 3, 3, 4]
...
+
+    if (array[0] === array[1,2] && array[3] === array[4] || array[0] === array[2,3] && array[1] === array[4] || array[0] === array[3,4] && array[1] === array[2])
+        
+        return true
+
+    else{
+
+        return false
+
+    }
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:172]: My main feedback is whitespace issues. Code looks great!

File: interview-questions.md [hunk 4]
```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
...
 
 
 
 3. What is Ruby on Rails?
 
-  Your answer:
+  Your answer: Ruby on Rails is an open source framework. Ruby on Rails helps build web applications in a more seamless way. 
 
-  Researched answer:
+  Researched answer: Ruby on Rails is a framework which contains a collection of code, tools and utilities that give a specific structure to work with when writing code. Rails helps build websites and contains something called Active Record which helps you read, create and update records in a database. Some exmaples of Ruby on Rails Applications are github, shopify, ask.fm, kickstarter, twitch, instacart and soundcloud.
```
Comment (WalkerTexas-coder) [inline interview-questions.md:28]: Great answer!

**Review by WalkerTexas-coder — APPROVED** [review]
Andrew this all looks great! I think you should get in the habit of cleaning up your whitespace as that will not be something that flies at a workplace. But honestly your code and approach to problem solving is top notch!

File: code-challenges.test.js (hunk 3 above)
Comment (lundrew) [reply code-challenges.test.js:172]: I'll keep that in mind. Thanks!
